adaptive_navigation/lib/scaffold.dart

```diff
@@ -68,52 +63,52 @@ class _AdaptiveNavigationScaffoldState
   @override
   Widget build(BuildContext context) {
     final NavigationTypeResolver navigationTypeResolver =
-        widget.navigationTypeResolver ?? _defaultNavigationTypeResolver;
+        this.navigationTypeResolver ?? _defaultNavigationTypeResolver;
```

Comment:
I don't think `this.` is required here.

Reply:
+1

Reply:
@johnpryan 

Thanks for the review.

`this.` is needed because local variable name `navigationTypeResolver` and parameter name `navigationTypeResolver` is same.

Omitting `this.` by changing local variable name is possible but I kept it and distinct names by using `this.`.